automatic/jenkins/ReadMe.md

```diff
@@ -0,0 +1,44 @@
+# <img src="https://cdn.jsdelivr.net/gh/pauby/chocopackages@ae92e032839a27a3633db7018593b075df44269f/icons/jenkins.png" width="48" height="48"/> [Jenkins](https://chocolatey.org/packages/jenkins)
+
+This is the LTS version of Jenkins.
+
+Jenkins is an open source automation server which enables developers around the world to reliably build, test, and deploy their software.
+
+## Features
+
+* Continuous Integration and Continuous Delivery
+As an extensible automation server, Jenkins can be used as a simple CI server or turned into the continuous delivery hub for any project.
+
+* Easy installation
+Jenkins is a self-contained Java-based program, ready to run out-of-the-box, with packages for Windows, Mac OS X and other Unix-like operating systems.
+
+* Easy configuration
+Jenkins can be easily set up and configured via its web interface, which includes on-the-fly error checks and built-in help.
+
+* Plugins
+With hundreds of plugins in the Update Center, Jenkins integrates with practically every tool in the continuous integration and continuous delivery toolchain.
+
+* Extensible
+Jenkins can be extended via its plugin architecture, providing nearly infinite possibilities for what Jenkins can do.
+
+* Distributed
+Jenkins can easily distribute work across multiple machines, helping drive builds, tests and deployments across multiple platforms faster.
```

Comment:
Don't leave a space between the list items, also indent the second line in the list.

```suggestion
* Continuous Integration and Continuous Delivery
  As an extensible automation server, Jenkins can be used as a simple CI server or turned into the continuous delivery hub for any project.
* Easy installation
  Jenkins is a self-contained Java-based program, ready to run out-of-the-box, with packages for Windows, Mac OS X and other Unix-like operating systems.
* Easy configuration
  Jenkins can be easily set up and configured via its web interface, which includes on-the-fly error checks and built-in help.
* Plugins
  With hundreds of plugins in the Update Center, Jenkins integrates with practically every tool in the continuous integration and continuous delivery toolchain.
* Extensible
  Jenkins can be extended via its plugin architecture, providing nearly infinite possibilities for what Jenkins can do.
* Distributed
  Jenkins can easily distribute work across multiple machines, helping drive builds, tests and deployments across multiple platforms faster.
```